UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/Configuration/CacheConfiguration.cs

```diff
@@ -7,6 +7,7 @@ public class CacheConfiguration
         public string CacheStorageAccountKey { get; set; }
         public string FssWeeklyBatchSearchTableName { get; set; }
         public string FssCacheResponseTableName { get; set; }
+        public string MsiBannerNotificationTableName { get; set; }
```

Comment:
Although it's a new table alongside the existing cache tables, it's not really part of the cache mechanism and therefore shouldn't be defined in the cache configuration?

Reply:
Thanks for highlighting, done needed changes.
Moved the MsiBannerNotificationTableName to the cache configuration.